PWGCF/FEMTOSCOPY/macros/AddTaskFemtoNanoGrandma.C

```diff
@@ -1165,7 +1204,7 @@ AliAnalysisTaskSE *AddTaskFemtoNanoGrandma(bool fullBlastQA = false, int phiSpin
   task->SetCorrelationConfig(config);
   mgr->AddTask(task);
 
-  TString addon = "PL";
+  TString addon = "HM";
```

Comment:
this is not going to work. Here make a unique specifier for your analysis ("BBar" or so), otherwise we won't be able to run multiple analyses at the same time

Reply:
... and maybe for the same reason change the name of the task
  AliAnalysisTaskNanoLoton* task = new AliAnalysisTaskNanoLoton("**femtoLoton**");